tests/integration/chat/api/test_message_service.py

```diff
@@ -0,0 +1,464 @@
+"""Unit tests for the MessageService."""
```

Comment:
JFYI: Skipped reviewing this in depth but looked like that this provides basic testing of the changes - like :)